CHANGELOG.md

```diff
@@ -40,8 +40,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - EQIP json GET endpoints. [#35](https://github.com/policy-design-lab/pdl-api/issues/35)
-- Title II CSP json GET endpoints. [#45](https://github.com/policy-design-lab/pdl-api/issues/45)
+- Title I Commodities json GET endpoints. [#40](https://github.com/policy-design-lab/pdl-api/issues/40)
```

Comment:
It looks like a merge may have updated a previous entry. Please fix the CHANGELOG entry and move this to the unreleased section.

Reply:
fixed it